### ISSUE TITLE
Suppress/skip reporting

### ISSUE THREAD

**Issue by Zodiase** [reviewdog/action-eslint#67]
I wanted to run eslint and report for all files to have a good idea of how many issues are remaining. However, there are too many issues. So I want to run two checks, one reports only the issue with the changes, while the other checks for everything but does not report.

Now I have something like this (skipping the unimportant things): 
```
jobs:
  eslint-react--full:
    steps:
      - uses: reviewdog/action-eslint@v1
        with:
          eslint_flags: 'src/'
          filter_mode: nofilter
          fail_on_error: true
  eslint-react--diff:
    steps:
      - uses: reviewdog/action-eslint@v1
        with:
          reporter: github-pr-check
          eslint_flags: 'src/'
          filter_mode: diff_context
          fail_on_error: false
```

Now I realize `reporter` has a default value of `github-pr-review` so me not specifying anything doesn't really do what I want.

So is it possible to suppress the reporting? Do you have other suggestions?

Also I noticed that even with this config, I'm still getting check annotations on files I didn't change. What's going on?

**Comment by Zodiase** [comment]
> Also I noticed that even with this config, I'm still getting check annotations on files I didn't change. What's going on?

I then changed `filter_mode` from `diff_context` to `file` and the check annotations on unchanged files are gone. Only the annotations for the modified files are left.

**Comment by haya14busa** [comment]
Maybe you can tweak your github action workflow to run reviewdog with whole code base only when event is push to master/main.
